src/driver.ts

```diff
@@ -42,6 +42,21 @@ export interface Device {
   type: DeviceType;
 }
 
+const isLitraDevice = (device: HID.Device): boolean => {
+  return (
+    device.vendorId === VENDOR_ID &&
+    PRODUCT_IDS.includes(device.productId) &&
+    device.usagePage === USAGE_PAGE
+  );
+};
+
+const HIDDeviceToDevice = (hidDevice: HID.Device): Device => {
```

Comment:
Nit: I'd prefer to use a `camelCase` notation for function names.

```suggestion
const hidDeviceToDevice = (hidDevice: HID.Device): Device => {
```